### PR TITLE
Flex docs - app, estimate & actual scan data

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206672453142664

### CODE REVIEW

File: docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+sidebar_label: Estimate and Actual Scan Data
+description: Learn about the estimate and actual scan data for Flex pricing.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::important
+Search modifier `_datatier` is not supported for any Flex query.
+:::
+
+When you enter a query that runs against the Flex pricing, Sumo Logic estimates and displays the amount of data that is scanned in Flex to return the search results. This detail is vital as in Flex you will pay for whatever amount of data that are scanned to complete the query.
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md:13]:
```suggestion
When you run a query against Flex pricing, Sumo Logic will estimate and display the amount of data that is scanned in Flex to return the search results. This detail is vital; with Flex pricing, you're charged for the amount of data that are scanned to complete the query. 
```

File: docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::important
+Search modifier `_datatier` is not supported for any Flex query.
+:::
+
+When you enter a query that runs against the Flex pricing, Sumo Logic estimates and displays the amount of data that is scanned in Flex to return the search results. This detail is vital as in Flex you will pay for whatever amount of data that are scanned to complete the query. 
+
+You can view this detail by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" />. A popup appears on click, which displays the estimated scan data breakup for the chargeable tiers like Infrequent and Flex.
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md:15]:
```suggestion
You can view this detail by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" />. A popup will appear that displays the estimated scan data breakup for the chargeable tiers, like Infrequent and Flex. 
```

File: docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Search modifier `_datatier` is not supported for any Flex query.
+:::
+
+When you enter a query that runs against the Flex pricing, Sumo Logic estimates and displays the amount of data that is scanned in Flex to return the search results. This detail is vital as in Flex you will pay for whatever amount of data that are scanned to complete the query. 
+
+You can view this detail by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" />. A popup appears on click, which displays the estimated scan data breakup for the chargeable tiers like Infrequent and Flex. 
+
+The example below shows the estimate of how much Flex data will be scanned for a query in the scope.<br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/estimated-scanned-data.png')} alt="estimated-scanned-data" style={{border:'1px solid gray'}} width="800" /> 
+After you run a query, you can see the actual Flex data was scanned for a query in the scope.<br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/actual-scanned-data.png')} alt="actual-scanned-data" style={{border:'1px solid gray'}} width="800" />
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md:18]:
```suggestion
The example below shows the estimate of how much Flex data will be scanned for a query in the scope.<br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/estimated-scanned-data.png')} alt="estimated-scanned-data" style={{border:'1px solid gray'}} width="800" /> 

After you run a query, you can see the actual Flex data was scanned for a query in the scope.<br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/actual-scanned-data.png')} alt="actual-scanned-data" style={{border:'1px solid gray'}} width="800" />
```

File: docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+:::note
+If there is no pay per scan tier scanned, an error message will be displayed in the **Scan Estimates** popup.
+:::
+
+When you click in the status area under the histogram, a popup with more detailed information appears. <br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/scan-details.png')} alt="scan-details" style={{border:'1px solid gray'}} width="500" />
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md:24]:
```suggestion
When you click in the status area under the histogram, you'll see a popup with containing detailed information. <br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/scan-details.png')} alt="scan-details" style={{border:'1px solid gray'}} width="500" />
```

File: docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Sumo logic will charge you every time you scan the data. So scheduling the search will help you to reduce some prices. You can also see the data scanned in **Time range for scheduled searches** for the selected run frequency while creating/modifying the scheduled search. <br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/scan-schedules-search.png')} alt="scan-schedules-search" style={{border:'1px solid gray'}} width="450" />
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data.md:26]:
```suggestion
Sumo Logic charges you every time you scan your data. Scheduling your searches can help you reduce costs. When you're creating or modifying a scheduled search, you can see your data scanned in **Time range for scheduled searches** for the selected run frequency . <br/><img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/scan-schedules-search.png')} alt="scan-schedules-search" style={{border:'1px solid gray'}} width="450" />
```

File: docs/manage/partitions-data-tiers/flex-pricing/index.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+With Sumo Logic Flex, you gain an efficient, centralized log analytics framework capable of managing enterprise-wide cloud-scale log ingestion without cost concerns. Consolidate all your data streams — application, infrastructure, security — into a single platform with unlimited user access, fostering seamless collaboration.
+
+The more log data ingested, the sharper your analytics and ML/AI insights become. By eliminating ingest limitations and empowering an ML/AI-driven single source of truth for analytics, Flex enables DevOps and DevSecOps teams to troubleshoot faster, accelerate release velocity, and ensure reliable, secure digital experiences.
+
+:::note
+Effective March 5, Flex pricing is the default plan for new customers who sign up with Sumo Logic.
+
+Effective May 15, Flex will become available for existing customers, in parallel with the three data tiers (Frequent, Infrequent, and Continuous).
+:::
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/flex-pricing/index.md:17]:
```suggestion
import FlexPricing from '../../../reuse/flex-pricing.md';

<FlexPricing/>
```

File: blog-service/2020/12-31.md [hunk 7]
```diff
@@ -571,7 +571,7 @@ New - Sometimes good things come in threes. That's true of our [Data Tiers](/do
 
 The Infrequent Tier is a cost-effective, fully managed log analytics solution for high volume, infrequently accessed data. This tier allows you to aggregate, store, and analyze verbose data that you access infrequently, like debug, CDN, and load balancer logs---at a dramatically lower price point, starting at only 10 cents per GB ingested. 
 
-The Infrequent Tier is only available only in Enterprise Suite available with [Cloud Flex Credits](/docs/manage/manage-subscription/cloud-flex-credits-accounts) packaging. 
+The Infrequent Tier is only available only in Enterprise Suite available with [Sumo Logic Credits](/docs/manage/manage-subscription/sumo-logic-credits-accounts) packaging.
```
Comment (kimsauce) [inline blog-service/2020/12-31.md:574]:
```suggestion
The Infrequent Tier is only available only in Enterprise Suite available with [Sumo Logic Credits](/docs/manage/manage-subscription/sumo-logic-credits-accounts) packaging. 

:::note
Effective March 2024, Cloud Flex Credits is now Sumo Logic Credits.
:::
```

File: blog-service/2021/12-31.md [hunk 8]
```diff
@@ -177,7 +177,7 @@ You should definitely try out the new outlier support in monitors if you have cu
...
+New - We're making [Sumo Logic Credits subscriptions](/docs/manage/manage-subscription/sumo-logic-credits-accounts) available to self-service trial and free accounts with new monthly and annual subscriptions. Customers who upgrade to a self-service paid subscription from a Trial, POV Trial, or Sumo Free subscription will now upgrade directly to Essentials (credit-based), and have the option of either an annual or monthly subscription. For more information, see [Upgrade a Credits Account](/docs/manage/manage-subscription/upgrade-sumo-logic-credits-account).
```
Comment (kimsauce) [inline blog-service/2021/12-31.md:180]:
```suggestion
New - We're making [Cloud Flex Credits Subscriptions](/docs/manage/manage-subscription/sumo-logic-credits-accounts) available to self-service trial and free accounts with new monthly and annual subscriptions. Customers who upgrade to a self-service paid subscription from a Trial, POV Trial, or Sumo Free subscription will now upgrade directly to Essentials (credit-based), and have the option of either an annual or monthly subscription. For more information, see [Upgrade a Credits Account](/docs/manage/manage-subscription/upgrade-sumo-logic-credits-account).

:::note
Effective March 2024, Cloud Flex Credits is now Sumo Logic Credits.
:::
```

File: blog-service/2022/12-31.md [hunk 9]
```diff
@@ -483,7 +483,7 @@ New - We pleased to announce the availability of the [Carbon Black Cloud](/docs/
 ---
 ## June 7, 2022 (Manage)
 
-New - Want to upgrade or change your [Cloud Flex Credits Essentials](/docs/manage/manage-subscription/cloud-flex-credits-accounts) plan? Good news: now you can do it yourself. If you’re a Sumo Logic admin, you can use the **Manage Plan** page to change your plan period from monthly to annual, or from annual to monthly. You can also increase or decrease your account ingest levels and retention settings to meet your evolving requirements. For more information, see [Update an CloudFlex Credits Account](/docs/manage/manage-subscription/upgrade-cloud-flex-credits-account).
+New - Want to upgrade or change your [Cloud Flex Credits Essentials](/docs/manage/manage-subscription/sumo-logic-credits-accounts) plan? Good news: now you can do it yourself. If you’re a Sumo Logic admin, you can use the **Manage Plan** page to change your plan period from monthly to annual, or from annual to monthly. You can also increase or decrease your account ingest levels and retention settings to meet your evolving requirements. For more information, see [Update a Sumo Logic Credits Account](/docs/manage/manage-subscription/upgrade-sumo-logic-credits-account).
```
Comment (kimsauce) [inline blog-service/2022/12-31.md:486]:
```suggestion
New - Want to upgrade or change your [Cloud Flex Credits Essentials](/docs/manage/manage-subscription/sumo-logic-credits-accounts) plan? Good news: now you can do it yourself. If you’re a Sumo Logic admin, you can use the **Manage Plan** page to change your plan period from monthly to annual, or from annual to monthly. You can also increase or decrease your account ingest levels and retention settings to meet your evolving requirements. For more information, see [Update a Cloud Flex Credits Account](/docs/manage/manage-subscription/upgrade-sumo-logic-credits-account).

:::note
Effective March 2024, Cloud Flex Credits is now Sumo Logic Credits.
:::
```

**Review by himanshu219 — APPROVED** [review]
Flex app documentation looks good to me

**Pull request merged**